### PR TITLE
Create: auto set funding target specific to zero on first try

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2483** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2483" target="_blank">Graphite</a> 
    * **PR #2484** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2484" target="_blank">Graphite</a> 
      * **PR #2486** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2486" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2486?utm_source=stack-comment).

File: src/components/Create/components/pages/FundingTarget/FundingTargetPage.tsx [hunk 1]
```diff
@@ -75,6 +76,15 @@ export const FundingTargetPage: React.FC = () => {
                     label: t`Funding Target`,
                   }),
                 ])}
+                extra={
+                  <Space style={{ alignItems: 'start' }}>
+                    <QuestionCircleOutlined />
+                    <Trans>
+                      Any funds raised over this amount are considered
+                      'overflow' and are redeemable by your contributors.
```
Comment (tomquirk) [inline src/components/Create/components/pages/FundingTarget/FundingTargetPage.tsx:84]: maybe 'overflow' should get a `<i>`

File: src/components/Create/components/pages/FundingTarget/hooks/FundingTargetForm.ts [hunk 2]
```diff
@@ -17,36 +20,38 @@ export const useFundingTargetForm = () => {
...
-        amount: { amount: undefined, currency },
-      }
-    } else {
-      return {
-        targetSelection: 'specific',
-        amount: {
-          amount: fromWad(distributionLimit.amount),
+    if (distributionLimit) {
+      if (distributionLimit.amount.eq(MAX_DISTRIBUTION_LIMIT)) {
+        amount = { amount: '0', currency }
```
Comment (tomquirk) [inline src/components/Create/components/pages/FundingTarget/hooks/FundingTargetForm.ts:37]: **question** just checkin we def want to set amount to '0' when dist limit is infinite

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2486?utm_source=stack-comment).


Job Reference: LMkvkXFiFBF1D050qgTf

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2486?utm_source=stack-comment).


Job Reference: rBMJjVCwIZgfc3ZnVfFL

**Pull request merged**